WalletWasabi/Services/UpdateManager.cs

```diff
@@ -369,11 +349,5 @@ public void StartInstallingNewVersion()
 		}
 	}
 
-	public void Dispose()
-	{
-		UpdateChecker.UpdateStatusChanged -= UpdateChecker_UpdateStatusChangedAsync;
-
-		CancellationTokenSource.Cancel();
-		CancellationTokenSource.Dispose();
-	}
+	private record GithubResult(Version LatestClientVersion, string InstallerDownloadUrl, string InstallerFileName, string Sha256SumsUrl, string WasabiSigUrl);
```

Comment:
This looks to me more like a `ReleaseInfo` than a `GithuResult`, in fact there is nothing related to github there.